main.gd

```diff
@@ -7,9 +7,9 @@ const BASE_GAMEJOLT_API_URL = 'https://api.gamejolt.com/api/game/v1_2'
 
 export(String) var private_key
 export(String) var game_id
-export(bool) var verbose
+export(bool) var verbose=true
```

Comment:
Verbose should be false by default. It sometimes produces messages, which has sensitive data.